src/data_vis.ts

```diff
@@ -354,6 +355,17 @@ class DataVisModel {
         });
     }
 
+    private exportAllData() {
+        const warnMsg : string = "Warning the dataset is quite large and can take several minutes to download";
+        const fileName : string = reportInfo.rep_id + "_dataset.csv"
```

Comment:
This filename isn't used in the download.

Reply:
Fixed.